Remove unnecessarily overwriting default subchart values

fabiodmota opened the pull request:
## Description

Image for both applications were removed and it used the default on subcharts
There will be only needed to override when there are specific versions/tags to use example https://github.com/catenax-ng/tx-vas-country-risk-frontend/blob/helm-env/charts/country-risk/values-dev.yaml


https://github.com/eclipse-tractusx/vas-country-risk/issues/55

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

SebastianBezold commented:
Hi @fabiodmota, 

the workflow failure is actually right. Changes to the default `values.yaml` are changes to the overall Chart, so it should be released with a new version. In this case i would say it's a patch.

Error message from linting workflow: `country-risk => (version: "3.0.3", path: "charts/country-risk") > chart version not ok. Needs a version bump! `

SebastianBezold reviewed (changes requested):
Increasing the Chart version to `3.0.4` should be added

fabiodmota commented:
> ` [1b](/eclipse-tractusx/vas-country-risk/pull/59/commits/1b3dbe0d76cb3265d128a2d23756ba636839b7ef)`

Hi @SebastianBezold , rebased with eclipse main and pushed new commit with version increase https://github.com/eclipse-tractusx/vas-country-risk/pull/59/commits/1b3dbe0d76cb3265d128a2d23756ba636839b7ef

SebastianBezold commented:
> > ` [1b](/eclipse-tractusx/vas-country-risk/pull/59/commits/1b3dbe0d76cb3265d128a2d23756ba636839b7ef)`
> 
> Hi @SebastianBezold , rebased with eclipse main and pushed new commit with version increase [1b3dbe0](https://github.com/eclipse-tractusx/vas-country-risk/commit/1b3dbe0d76cb3265d128a2d23756ba636839b7ef)

Yes you have: https://github.com/eclipse-tractusx/vas-country-risk/actions/runs/7017393605/workflow?pr=59#L85
This version is only provided, if you use the `workflow_dispatch` event. You did not add a default, that would be used, when triggered by a PR event

fabiodmota commented:
> > > ` [1b](/eclipse-tractusx/vas-country-risk/pull/59/commits/1b3dbe0d76cb3265d128a2d23756ba636839b7ef)`
> > 
> > 
> > Hi @SebastianBezold , rebased with eclipse main and pushed new commit with version increase [1b3dbe0](https://github.com/eclipse-tractusx/vas-country-risk/commit/1b3dbe0d76cb3265d128a2d23756ba636839b7ef)
> 
> Yes you have: https://github.com/eclipse-tractusx/vas-country-risk/actions/runs/7017393605/workflow?pr=59#L85 This version is only provided, if you use the `workflow_dispatch` event. You did not add a default, that would be used, when triggered by a PR event

Noted and fixed with my last commit
